openpype/plugins/actions/open_file_explorer.py

```diff
@@ -0,0 +1,50 @@
+import os
+import platform
+
+from openpype.client import (
+    get_asset_by_name,
+)
+from openpype.pipeline import (
+    Anatomy,
+    LauncherAction,
+)
+from openpype.pipeline.template_data import (
+    get_asset_template_data,
+)
+
+
+class OpenTaskPath(LauncherAction):
+    name = "open_task_path"
+    label = "Open in File Browser"
+    icon = None
+    order = 500
+
+    def is_compatible(self, session):
+        """Return whether the action is compatible with the session"""
+        return "AVALON_ASSET" in session
+
+    def process(self, session, **kwargs):
```

Comment:
I don't know the background, but I do expect it should require task name, and it should use workfilte templates instead of expecting that each project has same hardcoded template.